pkg/projects/functions.go

```diff
@@ -74,3 +77,40 @@ func SendCommandSSH(ip *string, port *int, user, password, command string, foreg
 		return nil
 	}
 }
+
+//SendCommandSCP try to access SSH with timer and sends command
```

Comment:
Name does not reflect what the function is doing. It doesn't send a command, it sends a file. So I would name it `SendFileSCP`, for example.

And comment above also needs to be changed

Reply:
Fixed